Arduino-Bling/RobotBling/helpers.ino

```diff
@@ -0,0 +1,50 @@
+/*
+ * helper methods
+ */
+
+// control full strip brightness
```

Comment:
Not 'control'. 'Set strip brightness to 100%'

Reply:
That was my lazy programming... comment was *meant* to cover both methods :)